pom.xml

```diff
@@ -56,6 +56,8 @@
     <maven-war-plugin.version>3.0.0</maven-war-plugin.version>
 
     <booster-common.version>2</booster-common.version>
+
+    <fabric8.generator.from>registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift</fabric8.generator.from>
```

Comment:
This is the Red Hat public Docker registry. Feel free to try :-)

And I think we want to use it by default, per https://github.com/openshiftio/appdev-planning/issues/36